ZipBuilder/Sources/sem-versions/PackageReader/PackageData.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+enum PackageType {
+  case cocoapods
+  case swiftPM
+}
+
+// TODO: Consider using a specific type to represent semantic version
+// like https://github.com/mxcl/Version
+typealias PackageVersion = String
+
+// TODO: Use something like https://github.com/kylef/PathKit
+typealias Path = String
+
+struct PackageData {
+  var name: String
+  var type: PackageType
+  var version: PackageVersion
+
+  var publicHeaderPaths: [Path]
+  var sourceFilePaths: [Path]
+
+  // TODO: Add necessary fields.
+}
+
+extension PackageData {
+  static func undefined() -> PackageData {
```

Comment:
I may have missed it but where is this used?

Reply:
It was an intermediate helper. Removed.